pkg/controller/vsphere/reconciler_test.go

```diff
@@ -971,6 +980,29 @@ func TestDelete(t *testing.T) {
 		},
 	}
 
+	testConfig := fmt.Sprintf(testConfigFmt, url[1])
```

Comment:
Should probably check the length of URL before taking the second element out of it?